AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/AWSIncomingEventReconciliationQueue.swift

```diff
@@ -87,6 +87,13 @@ final class AWSIncomingEventReconciliationQueue: IncomingEventReconciliationQueu
             self.eventReconciliationQueueTopic.send(.mutationEvent(event))
         }
     }
+
```

Comment:
Adding this here because I can't comment above: s/onRecieve*/onReceive*/g

Reply:
Updated